src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/ua/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple_1/"
+              class="nav_link is-active"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Instead of 'nav_link is-active', consider using 'nav__link nav__link--active' to follow BEM naming conventions.